Components/cards/cards.css

```diff
@@ -0,0 +1,182 @@
+body {
+  font-family: sans-serif;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.card-basic {
+  width: 20rem;
+  border-radius: 4px;
+  box-shadow: 0 2px 1px -1px rgb(0 0 0 / 20%), 0 1px 1px 0 rgb(0 0 0 / 14%),
+    0 1px 3px 0 rgb(0 0 0 / 12%);
+  background-color: white;
+}
+
+.card-img {
+  width: 100%;
+  height: 12rem;
+  object-fit: cover;
+  border-radius: 4px 4px 0 0;
+}
+
+.card-heading {
+  margin-left: 12px;
+  margin-bottom: 4px;
+  letter-spacing: 0.025rem;
+  color: #393d3d;
+}
+
+.card-subHeading {
+  margin-left: 12px;
+  letter-spacing: 0.03rem;
+  color: #6a6e6e;
+  font-weight: 500;
+}
+
+.card-description {
+  margin-left: 12px;
+  margin-top: 1rem;
+  letter-spacing: 0.025rem;
+  color: #838686;
+  font-weight: lighter;
+  font-size: 0.9rem;
+  line-height: 1.2rem;
+}
+
+.card-footer {
+  margin: 1.6rem 0;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.card-btn {
+  border: 0;
+  background-color: transparent;
+  cursor: pointer;
+  letter-spacing: 0.04rem;
+  color: #32c1cd;
+  font-weight: 600;
+}
+
+.heart-icon {
+  width: 20px;
+  height: 20px;
+}
+
+/* card text over media */
+
+.card-text-overMedia {
+  position: relative;
+}
+
+.heading-subHeading-overMedia-container {
```

Comment:
i think there is no class with .heading-subHeading-overMedia-container  if not needed u can remove this

Reply:
Yaah actually its for different version of card , which I will be using in future, currently I am not using it , thats why you cant see it in html file